makerdao/governance-portal-v2
# Fix/UI elems, gas price indicator, mkr weight for delegate owners and change tooltips

### Link to Shortcut ticket:
https://app.shortcut.com/dux-makerdao/story/1585/replace-sun-icon-with-moon-for-darkmode-toggle-in-menu
https://app.shortcut.com/dux-makerdao/story/1855/fix-gas-price-indicator
https://app.shortcut.com/dux-makerdao/story/1787/fix-mkr-weight-display-of-addresses-that-own-delegate-contracts
https://app.shortcut.com/dux-makerdao/story/1783/remove-your-vote-subheader-on-executive-detail-page
https://app.shortcut.com/dux-makerdao/story/1848/fix-overlap-of-embedded-media-on-poll-detail-page
https://app.shortcut.com/dux-makerdao/story/1836/deprecate-reach-ui-by-replacing-long-tooltips-with-dialogs

And also, added a skeleton component for some delegate balances. I think there's no ticket for it yet but it was discussed on Discord a couple of days ago: https://discord.com/channels/836646301986848768/882969853391163392/1064494148192833546

### Steps for testing:
1. On the header menu, switch between dark and light mode and make sure the icon updates accordingly.
2. On any of the pages, check the gas price indicator is showing in the header bar and that the gas price estimation is correct.
3. Delegate owner addresses used to show some MKR balance corresponding to the MKR in their delegate contract, e.g. https://vote.makerdao.com/address/0xa3f0abb4ba74512b5a736c5759446e9b50fda170#account-details, check that this is not the case anymore.
4. Verify that the `Your Vote` header is not present anymore on the voting section of any executive.
5. There's a couple of old polls that have large images on the poll details, e.g. https://vote.makerdao.com/polling/QmPEM83s#poll-detail, their images should adjust to the container size now.
6. Go to the delegates section and check that both `MKR delegated by you` and `Total MKR delegated` display skeleton components when loading instead of 0.
7. Tooltips changed for modals:
7.1. Polling contracts v1 and v2
7.2. Delegate participation and communication stats (on delegate cards of delegates section and on delegate metrics section of a delegate page)
7.3. Poll types (on each poll card of the polling section)
7.4. Finally, changed a couple of tooltips for them to be multi-line instead of one long line.
